packages/web/components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import Logo from './Logo';
+import NavigationButtons from './NavigationButtons';
+
+interface IProps {
+  height: number;
+}
+
+const Header: React.FC<IProps> = React.memo(({ height }) => {
+  const styles = {
```

Comment:
Wyciągnięcie tych styli to osobnej zmiennej jest niekonieczne, bo używasz ich tylko raz. Lepiej będzie po prostu umieścić je w elemencie: `<div css={{ ... my css selectors... }}`